Rmd/extra_stats_report.Rmd

```diff
@@ -1,24 +1,24 @@
 ---
 params:
   projectdir: null # Use when loading. If null, here::here() will be used. Can also hard code if for some reason you need that
-  project_title: "Default"    # Change depending on your project name
+  project_title: "GeneTox21"    # Change depending on your project name
```

Comment:
Yeah in my opinion these changes shouldn't be tracked... It just happens sometimes by mistake. Normally I avoid staging those changes, but it's hard to catch every time. I wonder if there's a way to "mask" parts of a file from being tracked? not that I'm aware of... also would be problematic when we DO need to update aspects...

Reply:
Yeah I know. It gets harder with having them in each file too. I thought that maybe setting them to something obviously default (maybe null for everything?) would make it easier to keep these untouched manually? Or at least make it low-effort to fix.

Reply:
Null for everything would probably be fine - I think that works as long as everything is defined in the config.yaml. It is possible some aren't (you can't pass params that aren't in the Rmd, but you can have some missing from the config and it will use the default. though I don't think this is the case for us, since they were actually running fine when you passed in the list without have any params predefined... still not sure if that is a bug or a feature, since it screwed up parallel runs... probably something to do with how it's a read-only list in RMarkdown but I digress)